internal/cmds/cmds.go

```diff
@@ -76,6 +76,12 @@ var (
 		cs: newCommandSlice([]string{"UNSUBSCRIBE", "+sentinel", "+slave", "-sdown", "+sdown", "+switch-master", "+reboot"}),
 		cf: noRetTag,
 	}
+
+	// DiscardCmd is predefined DISCARD
+	DiscardCmd = Completed{
+		cs: newCommandSlice([]string{"DISCARD"}),
+		cf: noRetTag,
```

Comment:
```suggestion
```

We can’t set the noRetTag on it. The tag will trigger a special handling for PUBSUB commands.